app/assets/stylesheets/config/_fonts.scss

```diff
@@ -14,3 +14,37 @@ $headers-font: "Staatliches", "Helvetica", "sans-serif", cursive;
 //        font-url('FontFile.ttf') format('truetype')
 // }
 // $my-font: "Font Name";
+
+html {
+  font-size: 13px !important;
```

Comment:
In general, the !important rule should only be used as a last resort, and if you find yourself using it too often you should look over the CSS and try to see why it's needed in the first place.

Reply:
Yeah am aware - it was to override the bootstrap sizing, figured seeing as these lie within our master fonts file it makes at least some sense.